### PR TITLE
how to enable places events forwarding to edge

### PR DESCRIPTION
**Pull request by PravinPK**
Included a new page under Places Tab to demonstrate - "Steps to configure Places Events forwarding to Experience platform"

Highlighted the feature capabilities and the steps to enable this feature

### CODE REVIEW

File: src/pages/documentation/places/places-to-aep.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Steps to configure Places Event forwarding for Experience Platform
```
Comment (imiron-git) [inline src/pages/documentation/places/places-to-aep.md:1]: Places Service event forwarding to Adobe Experience Platform

**Review by imiron-git — CHANGES_REQUESTED** [review]

File: src/pages/documentation/places/places-to-aep.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Hit confirm and finally, your schema's **placeContext** field group should look like:
+
+![Final Schema Structure](./assets/index/final_schema_structure.png)
+
+## Datastream Configuration
+
+* In the browser, navigate to the [Data Collection UI](https://experience.adobe.com/#/data-collection).
+* From the left panel, select **Datastreams** and from the list select the datastream that is configured for your application. For more details about configuring datastream, please read [this section](../getting-started/configure-datastreams.md).
+* Find **Adobe Experience Platform** service, click on `...` and choose **Edit**
+![Choose AEP Service](./assets/index/datastream_choose_aep_services.png)
```
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:56]: "The Adobe Experience Platform is highlighted."

File: src/pages/documentation/places/places-to-aep.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+* Save your Schema
+Hit confirm and finally, your schema's **placeContext** field group should look like:
+
+![Final Schema Structure](./assets/index/final_schema_structure.png)
+
+## Datastream Configuration
+
+* In the browser, navigate to the [Data Collection UI](https://experience.adobe.com/#/data-collection).
+* From the left panel, select **Datastreams** and from the list select the datastream that is configured for your application. For more details about configuring datastream, please read [this section](../getting-started/configure-datastreams.md).
+* Find **Adobe Experience Platform** service, click on `...` and choose **Edit**
```
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:55]: "... select `...`, and choose **Edit**."

File: src/pages/documentation/places/places-to-aep.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+Please remove `AJO Push Tracking dataset`, if it exist in the list of Event Dataset in your Datastream. This is a system dataset and is not meant to be configured in the Datastream.
+
+## Data Collection Tag Configuration
+
+* In the browser, navigate to the [Data Collection UI](https://experience.adobe.com/#/data-collection).
+* From the left panel, select **Tags** tab and from the list select the Tag that is configured for your application.
+* From the left panel, select **Extensions** tab.
+* Update the Places Launch Card to 1.3.0 or greater. No Additional configuration is required.
+* Make sure **Adobe Expereience Edge Network**, **Identity** and **Consent** extensions are installed.
+* Make sure **Adobe Expereience Edge Network** is configured with appropritate datastream which has dataset with **placecontext** field group enabled in their schema.
```
Comment (emdobrin) [inline src/pages/documentation/places/places-to-aep.md:71]: typo x 2: Experience Edge Network
typo: appropriate

File: src/pages/documentation/places/places-to-aep.md (hunk 1 above)
Comment (emdobrin) [reply src/pages/documentation/places/places-to-aep.md:1]: You need to also update the gatsy-config file to include this file in the left nav menu

File: src/pages/documentation/places/places-to-aep.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+![Final Schema Structure](./assets/index/final_schema_structure.png)
+
+## Datastream Configuration
+
+* In the browser, navigate to the [Data Collection UI](https://experience.adobe.com/#/data-collection).
+* From the left panel, select **Datastreams** and from the list select the datastream that is configured for your application. For more details about configuring datastream, please read [this section](../getting-started/configure-datastreams.md).
+* Find **Adobe Experience Platform** service, click on `...` and choose **Edit**
+![Choose AEP Service](./assets/index/datastream_choose_aep_services.png)
+* Make sure you have set up the Dataset with placeContext field group enabled Schema as one of your Event Datasets in the Datastream configuration (Be it primary or not).
```
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:57]: "Make sure you have set up the dataset with the `placeContext` field-group-enabled schema as one of your event datasets in the datastream configuration. You'll need to do this whether it is your primary dataset or not."

File: src/pages/documentation/places/places-to-aep.md
Comment (emdobrin) [inline src/pages/documentation/places/places-to-aep.md:1]: Would it help to also document the xdm event types expected when enabling the forwarding as well as how the event type+poiid field can be used after everything is set up?

File: src/pages/documentation/places/places-to-aep.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+![Ensure correct dataset is present](./assets/index/datastream_ensure_correct_dataset.png)
+* Make sure **Adobe Journey Optimizer** check box is enabled.
```
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:59]: "Make sure the **Adobe Journey Optimizer**..."

File: src/pages/documentation/places/places-to-aep.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+<InlineAlert variant="warning" slots="text"/>
+Please remove `AJO Push Tracking dataset`, if it exist in the list of Event Dataset in your Datastream. This is a system dataset and is not meant to be configured in the Datastream.
```
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:62]: "Please remove the `AJO Push Tracking dataset` if it exists in the list of event datasets in your datastream."

Also, we need a newline between the InlineAlert and the text.

File: src/pages/documentation/places/places-to-aep.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+## Data Collection Tag Configuration
+
+* In the browser, navigate to the [Data Collection UI](https://experience.adobe.com/#/data-collection).
+* From the left panel, select **Tags** tab and from the list select the Tag that is configured for your application.
+* From the left panel, select **Extensions** tab.
+* Update the Places Launch Card to 1.3.0 or greater. No Additional configuration is required.
```
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:69]: "No additional configuration is required."

File: src/pages/documentation/places/places-to-aep.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+* Make sure **Adobe Expereience Edge Network**, **Identity** and **Consent** extensions are installed.
```
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:70]: "Make sure the Adobe Experience Edge Network, Identity,..."

File: src/pages/documentation/places/places-to-aep.md (hunk 4 above)
Comment (calebk1) [inline src/pages/documentation/places/places-to-aep.md:71]: "... configured with the appropriate datastream which has a dataset with the `placeContext` field group..."

**Review by calebk1 — COMMENTED** [review]
Some comments!

Also, make sure to add this to the `gatsby-config.js`, otherwise this document won't be visible.

File: src/pages/documentation/places/places-to-platform.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
+# Places Service event forwarding to Adobe Experience Platform
+
+## Capablities
```
Comment (emdobrin) [inline src/pages/documentation/places/places-to-platform.md:3]: typo: Capabilities

File: src/pages/documentation/places/places-to-platform.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+1. Enriches Adobe Journey Optimizer's ability to engage with users based on location entry and exit triggers.
+2. Allows creating audiences based on points of interest (POI) visits.
+
+## Mobile SDK
+
+Upgrade your Mobile SDK to following Places extension version
```
Comment (emdobrin) [inline src/pages/documentation/places/places-to-platform.md:10]: Upgrade the Mobile SDK to the following Places extension version:

**Review by calebk1 — COMMENTED** [review]
Looks good! Just a couple of comments.

**Review by emdobrin — COMMENTED** [review]
looks good with few comments opened

File: gatsby-config.js [hunk 12]
```diff
@@ -334,7 +334,7 @@ module.exports = {
 
 
             {
-                title: "Adobe Experience Platform Location Service",
+                title: "Places Service, powered by Adobe Experience Platform",
```
[the remark this answers is not in the document]
Comment (PravinPK) [reply gatsby-config.js:337]: I will check with Ivan on this again. It seems "Places Service, powered by Adobe Experience Platform" is the offical name for Places

**Review by calebk1 — APPROVED** [review]
LGTM!

**Review by emdobrin — APPROVED** [review]

**Pull request merged**